src/test/resources/org/rust/ide/intentions/fixtures/extract_inline_module/before/main.rs

```diff
@@ -0,0 +1,6 @@
+mod foo {
```

Comment:
in another test case the caret not focus on `mod foo {` and there should no `ExtractInlineMod` there.